examples/receive_all/main.cc

```diff
@@ -0,0 +1,114 @@
+#include <zendar/api/api.h>
+
+#include <glog/logging.h>
+#include <gflags/gflags.h>
```

Comment:
I see that we're using `cout` everywhere in this file. Is there a reason to include `<glog/logging.h> here?

Reply:
not really, I can clean that up